react-app-rewired/index.ts

```diff
@@ -232,7 +232,15 @@ const reactAppRewireConfig = {
                     }
                     const definitions = Object.fromEntries(
                       Object.entries(env)
-                        .filter(([k]) => !k.startsWith('REACT_APP_'))
+                        .filter(
+                          ([k]) =>
+                            // Keep only REACT_APP_* env var entries we are using in packages. We *cannot* remove these, else
```

Comment:
Do we even need to filter out anymore, and can we simply keep *all* env vars, to make package env var usages more futureproof? Maybe.

But this is a webpack config, fragile by nature and it's not worth the huge risk in case something goes wrong, so only made sure to exclude out the ones we currently consume in unchained-client